packages/plugin-input-exercise/src/editor.tsx

```diff
@@ -98,24 +118,17 @@ export function InputExerciseEditor(
               <AddButton onClick={() => state.answers.insert()}>
                 Antwort hinzufügen...
               </AddButton>
-              <hr />
-              Wähle den Antworttyp:
-              <select
-                onChange={(event: React.ChangeEvent<HTMLSelectElement>) =>
-                  state.type.set(translateDataName(event.target.value))
-                }
-                value={translateDataType(state.type.value)}
-              >
-                {R.map(dataType => {
-                  return (
-                    <option key={dataType.name} value={dataType.name}>
-                      {dataType.name}
-                    </option>
-                  )
-                }, types)}
-              </select>
+              {!props.renderIntoExtendedSettings ? (
```

Comment:
ffr: if `renderIntoExtendedSettings` doesn't exist, we render `Controls` only if its in focus.
if it does exist, we render it independently from focus, because plugins loose focus when the Overlay gets triggered on the containing `rows` plugin....